docs/life_of_a_godwoken_transaction.md

```diff
@@ -144,38 +127,38 @@ This data structure contains 2 parts of information:
 
-Custodian cells contain all the tokens that are managed internally in Godwoken. This [transaction](https://explorer.nervos.org/aggron/transaction/0x05fb7fb33b092272a4e259688e21bd1c78320bda641b062a755af9ec2849223c) contains a custodian cell in its output cell #2. Like a deposit cell, a custodian cell is represented by its lock script:
+Custodian cells contain all the tokens that are managed internally in Godwoken. This [transaction](https://pudge.explorer.nervos.org/transaction/0x6c627c00d215ddc2dc21b6b40c03be8c14d4b7fca06e4d84b97e4bb526e5b135) contains a custodian cell in its output cell #2. Like a deposit cell, a custodian cell is represented by its lock script:
 
 ```
 {
-    "code_hash": "0x3c5551b6cce4d6873a636684f26fae173be653bf2122997b7023acad63a9fff1",
-    "args": "0x4fb03f323f32196cce0232d1398b6eba4e36654eb431b8c71aa45b745dbb1e0ab90000001000000091000000b10000008100000010000000300000007900000010291f3d4de6bbd924e2a6a0b5dbe96654540dc789a901c0b8826ae0f7786ea249000000100000003000000031000000000000000000000000000000000000000000000000000000000000000000000100140000009c11877e58c9822131ef09744856fd6b9ce2fa9e00a30200000000c05d5d69e73faa984f8619646fcf9aa971c72978604e191dab99d2d8588b68a6b13301000000000000",
+    "code_hash": "0x85ae4db0dd83f428a31deb342e4000af37ce2c9645d9e619df00096e3c50a2bb",
+    "args": "0x702359ea7f073558921eb50d8c1c77e92f760c8f8656bde4995f26b8963e2dd8e1000000100000003000000038000000595a1405676547c8ddc083f3413d58d65a1bb9275b5a923ec61dce4e572647cc4c6f000000000000a900000014000000340000009d000000a5000000e13f4a9c1642a6c59766eb5509580808bf3917bba104a616df0d207c93050e596900000010000000300000003100000007521d0aa8e66ef441ebc31204d86bb23fc83e9edc58c19dbb1b0ebe64336ec00134000000702359ea7f073558921eb50d8c1c77e92f760c8f8656bde4995f26b8963e2dd8e8ae579256c3b84efb76bbb69cb6bcbef1375f00813a0900000000c002000000",
     "hash_type": "type"
 }
 ```
 
-Like deposit cells, custodian cells have pre-determined `code_hash` and `hash_type` based on Godwoken deployments. The first 32 bytes in `args` contain the rollup type hash as well. What's different here, is that [CustodianLockArgs](https://github.com/nervosnetwork/godwoken/blob/v0.1.0/crates/types/schemas/godwoken.mol#L131) is used instead to fill the remaining part of `args`:
+Like deposit cells, custodian cells have pre-determined `code_hash` and `hash_type` based on Godwoken deployments. The first 32 bytes in `args` contain the rollup type hash as well. What's different here, is that [CustodianLockArgs](https://github.com/nervosnetwork/godwoken/blob/v1.1.0-beta/crates/types/schemas/godwoken.mol#L188) is used instead to fill the remaining part of `args`:
```

Comment:
Like deposit cells, custodian cells have pre-determined `code_hash` and `hash_type` based on Godwoken deployments. The first 32 bytes in `args` contain the rollup type hash as well. What is different here is that [CustodianLockArgs](https://github.com/nervosnetwork/godwoken/blob/v1.1.0-beta/crates/types/schemas/godwoken.mol#L188) is used to fill the remaining part of `args`: